static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -198,6 +220,78 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
     };
   }
 
+  get chartProps(): UsageChartProps {
+    const {dataCategory} = this.props;
+    const {error, errors, loading} = this.state;
+    const {
+      chartStats,
+      dataError,
+      chartDateInterval,
+      chartDateStart,
+      chartDateEnd,
+      chartDateUtc,
+      chartTransform,
+    } = this.chartData;
+
+    const hasError = error || !!dataError;
+    const chartErrors: any = dataError ? {...errors, data: dataError} : errors; // TODO(ts): AsyncComponent
+    const chartProps = {
+      isLoading: loading,
+      isError: hasError,
+      errors: chartErrors,
+      title: ' ', // Force the title to be blank
+      footer: this.renderChartFooter(),
+      dataCategory,
+      dataTransform: chartTransform,
+      usageDateStart: chartDateStart,
+      usageDateEnd: chartDateEnd,
+      usageDateShowUtc: chartDateUtc,
+      usageDateInterval: chartDateInterval,
+      usageStats: chartStats,
+    } as UsageChartProps;
+
+    return chartProps;
+  }
+
+  get cardMetadata() {
+    const {dataCategory, dataCategoryName, organization} = this.props;
+    const {total, accepted, dropped, filtered} = this.chartData.cardStats;
+
+    const cardMetadata: Record<string, ScoreCardProps> = {
```

Comment:
Converted the metadata from a list, to an object with keys as the stat type. This will let us pick specific cards to modify in GetSentry